code/__DEFINES/misc.dm

```diff
@@ -522,8 +522,7 @@ GLOBAL_LIST_INIT(pda_styles, sortList(list(MONO, VT, ORBITRON, SHARE)))
 #define NO_BUCKLE_LYING -1
 
 
-// timed_action_flags parameter for `/proc/do_atom`, `/proc/do_after_mob`, `/proc/do_mob` and `/proc/do_after`
-#define IGNORE_TARGET_IN_DOAFTERS (1<<0)
+// timed_action_flags parameter for `/proc/do_after_mob`, `/proc/do_mob` and `/proc/do_after`
 #define IGNORE_USER_LOC_CHANGE (1<<1)
 #define IGNORE_TARGET_LOC_CHANGE (1<<2)
 #define IGNORE_HELD_ITEM (1<<3)
```

Comment:
If you're removing a flag isn't it only good manners to recenter them at 0?

Reply:
i was raised in a barn